packages/rs-dpp/schema/meta_schemas/document/v0/document-meta.json

```diff
@@ -287,6 +287,39 @@
           }
         }
       ]
+    },
+    "documentActionTokenCost": {
+      "type": "object",
+      "properties": {
+        "contractId": {
+          "type": "array",
+          "contentMediaType": "application/x.dash.dpp.identifier",
+          "byteArray": true,
+          "minItems": 32,
+          "maxItems": 32
+        },
+        "tokenPosition": {
+          "type": "integer",
+          "minimum": 0
+        },
+        "amount": {
+          "type": "integer",
+          "minimum": 0
```

Comment:
minimum: 1
maximum: i64 max

Reply:
I'm not sure you could even put higher than a i64 max no matter what right?

Reply:
I'll put 2^48

Reply:
Why so? I think you can.